jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindowUI.kt

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.sqs.toolwindow
+
+import com.intellij.openapi.project.Project
+import com.intellij.ui.components.JBTabbedPane
+import software.aws.toolkits.jetbrains.services.sqs.Queue
+import software.aws.toolkits.resources.message
+import javax.swing.JPanel
+
+// Will add more parameters once window is populated
+class SqsWindowUI(private val project: Project, val queue: Queue) {
+    val mainPanel = JBTabbedPane().apply {
+        this.add(message("sqs.queue.message.sampling"), JPanel())
```

Comment:
nit: use the panes made above (they will need a JPanel added to them which we usually name `component`)